Conceptpower+Spring/src/main/java/edu/asu/conceptpower/root/DatabaseClient.java

```diff
@@ -296,6 +302,84 @@ public boolean match(ConceptList dict) {
      * lang.String)
      */
     @Override
+    public List<ConceptEntry> getAllEntriesFromList(String listname, int page, int pageSize, final String sortBy,
+            final int sortDirection) {
+        ConceptEntry entry = new ConceptEntry();
+        entry.setConceptList(listname);
+
+        Query wordNetQuery = wordnetCacheClient.query();
```

Comment:
What I'm saying is that this method ```getAllEntiresFromList``` is supposed to return all the entries of a concept list (now paginated), right? The entries that this method return can never be in the wordnet cache, because only concepts not in wordnet (or wrapped wordnet concepts) will be in a concept list and hence they will be stored in the concept database (not the wordnet cache). So, any concept in the wordnet cache simply cannot be in a concept list and therefore cannot be retrieved by this method. Does this make sense?